Makefile

```diff
@@ -98,7 +98,8 @@ test-all: $(TOX)
 	$(TOX)
 
 .PHONY: check
-check: clean test-all
+#check: clean test-all
```

Comment:
It's presence makes it a great deal easier to work on getting test-all working again, so yes.